tasks/oah-validate.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Checking the installed version

```

Comment:
Fails when sonarqube is not found. 
![image](https://user-images.githubusercontent.com/58501088/190390253-e1687deb-967e-4956-aa38-5a23ab483a86.png)

Suggesting to add some error handling.

Reply:
Improved the functioning of oah-validate.
Introduced error handling